### DISCUSSION TITLE
Scientific Writing

### DISCUSSION THREAD

**Discussion by ADParedes** [Open-Research-Program/HCOE#8] [Show and tell]
Hi!  I have a starting point for Scientific Writing Topic Concept. 
Please see the following slide deck and help me to make this more valuable to you all.  
[Quick Insight into Peer-review Publications and Technical Writing](https://docs.google.com/presentation/d/1gm3MMAVcYd2UcttzzDZqgPNDlXltDiyY/edit#slide=id.p2) 

Please comment on this Discussion thread things you would like to learn more about!

**Comment by spulido1** [comment]
I think this slide deck provides very helpful information. I am interested to know if there are any tips/tricks out there on how to paraphrase? Avoiding plagiarism is important and it can be a struggle sometimes to paraphrase literature that is already concise.

**Reply by Jdlopez0108** [reply]
> I think this slide deck provides very helpful information. I am interested to know if there are any tips/tricks out there on how to paraphrase? Avoiding plagiarism is important and it can be a struggle sometimes to paraphrase literature that is already concise.

I try to do this with organization and crafting a narrative. First, I'll make an outline. Then, I'll do the research and fill in my outline with data and statistics. Finally, you can weave the data into your narrative to support your unique voice. I find that it's easier to avoid plagiarizing this way because you're using your own words to tell the story and you just need to insert the data. 

For big concepts where you're citing more than just data points and statistics, I think it helps to give explicit credit and even include the story of that research and researcher in your narrative. That way, you are citing clearly and also providing compelling insight into the background of your research  (This was much of the feedback @ADParedes gave me on my presentation).

**Comment by mizquierdo2** [comment]
One of the most important takeaways I have from this deck is the fact that research does not have to be completely original, but new. Throughout this project and others I have been disappointed when I find papers that closely look at questions I have thought of but have since then learned that these projects would not have existed if it weren't for similar research projects leading them to add ideas to the same research question. I aim to do the same!

**Comment by jsejo** [comment]
I am most intrigued by the concept of technical writing. This slide deck helped me understand the importance of choosing words with purpose, ways in which to avoid reader confusion, and the benefit of omitting complex jargon. No additional questions at this time.

**Comment by bguill3** [comment]
I was surprised to find out that scientific writing should be concise and simple. Although I personally have not written any scientific research papers, some of the ones I have read seem to be quite difficult to understand and use a lot of jargon. I appreciate this advice as I will use it moving forward when writing papers myself. 

I would like more information on writing the background for a paper. The presentation mentioned only including what is relevant, but I find this challenging especially since I do not want it to seem that I did not thoroughly review the topic prior to writing the background. I am hoping this can be expanded on in detail a bit more.

**Comment by Jdlopez0108** [comment]
The points about Peer Review reminded me of the spider researcher who had many papers retracted around early 2020. The papers had duplicated data points and included numerous statistical anomalies. A large body of his work supporting social behavior and group selection in spiders had been retracted almost overnight. I was taking an Evolution course at the time and my professor, an ecologist, was in shock as to how it could've happened. This illustrates the point of bias in peer review really well, definitely an important thing to reflect on.

**Comment by jasson907** [comment]
This powerpoint was actually very helpful. My questions would revolve around asking how we should interpret feedback from PI's when you submit a writing and the entire writing comes back changed/ red all over. I've personally experienced this and my peers have too. Do PI's do this because they just care for a certain style of writing or is there a different reason?
